assets/js/vue/utils/template-switch.js

```diff
@@ -0,0 +1,26 @@
+const grp = {
+    status: 'Dataset Status',
+    fundingCycle: 'Grant Awards',
+    researchGroup:  'Projects',
+    fundingOrg: 'Funding Organizations'
```

Comment:
Here add another variable like: showFundingCycles, then you can use that function instead rather than checking if it's a template with a certain name.

Reply:
I added a section for flags, separate of the labels section and a method to get the flag value in whatever template that is in effect.